trusted-committer/04-uplevelling-community-members.asciidoc

```diff
@@ -1,41 +1,39 @@
+by encouraging them to tackle challenging tasks and helping them complete them. This is, in our opinion, a Trusted Committer's
+noblest responsibility, and it is rewarding for both Trusted Committer and
+contributor. We've heard Trusted Committers say that mentoring and
+seeing people level up their abilities more than makes up for the fact
+that they have less time to actually spend writing software.
 
-As mentioned in the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/03-keeping-the-community-healthy.asciidoc[previous section], learning and personal growth are reasons
-why people join and stick around in an InnerSource community. Upleveling their
-https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] is one of the most powerful tools Trusted Committers have at their disposal to
-increase the speed, output and longevity of their community. It is also one of
-the key arguments with which to convince management to allow their employees to
-participate in an InnerSource community, as that will make their employees more
-valuable to them, to the company overall and it will help retain top talent.
+As mentioned in the https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/trusted-committer/03-keeping-the-community-healthy.asciidoc[previous section], learning and personal growth are
+reasons people join and stick around in an InnerSource community.
+Upleveling their https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] is one of the most powerful tools Trusted Committers have
+at their disposal to increase the speed, output and longevity of their
+communities. It is also one of the key arguments with which to convince
+management to allow their employees to participate in an InnerSource
+community, as that will make their employees more valuable to 
+the company overall and help them retain top talent.
 
-In summary, Trusted Committers need to attract new https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] and level up their ability to
-make contributions.  This activity ultimately levels up the community's ability to create
-better software faster. They do so by communicating opportunities to make
-contributions and by helping and mentoring Contributors to grow
+In summary, Trusted Committers need to attract new https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/contributor/01-introduction-article.asciidoc[_Contributors_] and level up their
+ability to make contributions. This activity ultimately levels up the
+community’s ability to create better software faster. They do so by
+communicating opportunities to make contributions and by helping and
+mentoring contributors.
```

Comment:
Rest of the text *Contributors* was used a proper name and capitalized. Doing that here too.
I also liked the mentioning of professional growth in the original text so adding that to the commit suggestion too.

```suggestion
mentoring Contributors enabling them to grow.
```